guice-extension/src/main/java/name/falgout/jeffrey/testing/junit/guice/GuiceExtension.java

```diff
@@ -2,6 +2,7 @@
 
 import static java.util.stream.Collectors.toSet;
 import static org.junit.platform.commons.support.AnnotationSupport.findRepeatableAnnotations;
+import static org.junit.platform.commons.util.AnnotationUtils.findAnnotation;
```

Comment:
`AnnotationUtils` is an internal-only class: https://junit.org/junit5/docs/5.0.2/api/org/junit/platform/commons/util/AnnotationUtils.html

Use `AnnotationSupport.isAnnotated` instead: https://junit.org/junit5/docs/5.0.0/api/org/junit/platform/commons/support/AnnotationSupport.html#isAnnotated-java.lang.reflect.AnnotatedElement-java.lang.Class-